lib/structures/LavalinkClient.js

```diff
@@ -49,14 +49,17 @@ class LavalinkClient extends PlayerManager {
     }
 
     /**
-     * Gets the most ideal host based on specified guild region.
-     * @param {string} region The region of the guild
+     * Gets the most ideal host based on load.
      * @returns {string}
      */
-    getIdealHost(region) {
-        region = this.getIdealRegion(region);
-        const foundNode = this.nodes.find(node => node.ready && node.region === region);
-        return foundNode ? foundNode.host : this.nodes.filter(node => node.ready).first().host;
+    getIdealHost() {
```

Comment:
Could just make this return the node itself as you can always get the host from the node

Reply:
no it doesnt, its called idealNode and uses this function so that doesnt make sense.
Why not make this return the node so in idealNode it doesnt have to get the node again from the node and just get the node from here and not get it from the collection from the host from this function.